install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -409,9 +409,9 @@ bgpControlPlane:
   # -- SecretsNamespace is the namespace which BGP support will retrieve secrets from.
   secretsNamespace:
     # -- Create secrets namespace for BGP secrets.
-    create: true
+    create: false
     # -- The name of the secret namespace to which Cilium agents are given read access
-    name: cilium-bgp-secrets
+    name: kube-system
```

Comment:
Should the namespace be `cilium-secrets` instead of `kube-system`, I see other features like Gateway API etc use `cilium-secrets`.

Reply:
The way GatewayAPI or Ingress using `cilium-secrets` namespace is a bit different from others. They import Secrets from other namespaces into this `cilium-secrets` namespace. While the name is generic, it's dedicated to Ingress or GatewayAPI use cases. Also, it probably have more update churn than `kube-system`.

Reply:
Thanks @YutaroHayakawa makes sense